### DISCUSSION TITLE
Verbesserungsvorschlag Displayanzeige

### DISCUSSION THREAD

**Discussion by themanfrommoon** [tbnobody/OpenDTU#706] [Ideas]
Moin,

ich habe hier einen Verbesserungsvorschlag für die Anzeige auf den Displays:

Im Display soll ganz klein in einer Ecke angezeigt werden vor wie vielen Sekunden die Werte das letzte mal geholt wurden.
Um Platz zu sparen reichen wohl 3 Zeichen:
"-1" = vor 1 Sekunde
"-10"= vor 10 Sekunden
"-99" = vor 99 Sekunden
">99" = länger als 99 Sekunden her.....(bedeutet Fehler, Schlafmodus, Wechselrichter aus)
man könnte auch "---" oder "XXX" anzeigen bei Fehler oder Wechselrichter aus

Beste Grüße,
Chris

**Comment by uvcollector** [comment]
Sehr coole Sache! Das möchte ich auch gerne 😊
